dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.cdi4.1-servlet6.1.feature

```diff
@@ -0,0 +1,12 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.cdi4.1-servlet6.1
+visibility=private
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.cdi-4.1))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.servlet.internal-6.1))"
+-bundles=io.openliberty.cdi.4.0.internal.web, \
+ com.ibm.ws.cdi.web.jakarta
+-features=io.openliberty.jakarta.pages-4.0; apiJar=false
```

Comment:
While it has been present in previous verisons, I am not sure this line is required in this auto-feature as the public CDI feature already depends on pages. Just an observation, shouldn't make any difference if we keep it or remove it.

Reply:
I will note this down as a clean up item for a future PR.